Vagrantfile

```diff
@@ -164,55 +136,43 @@ Vagrant.configure(2) do |config|
   # The VirtualBox IP-address for the browser
   config.vm.network :private_network, ip: VM_IP
 
-  # Port forwarding for services running on VM, does not work on Windows
-  if not IS_WINDOWS
-    config.vm.network "forwarded_port", guest: 1080,  host: 1080,  auto_correct: true   # Mailcatcher
-    config.vm.network "forwarded_port", guest: 3306,  host: 3306,  auto_correct: true   # MySQL
-    config.vm.network "forwarded_port", guest: 5432,  host: 5432,  auto_correct: true   # PostgreSQL
-    config.vm.network "forwarded_port", guest: 10007, host: 10007, auto_correct: true   # Jenkins (development)
-  end
+  # Port forwarding for services running on VM
+  config.vm.network "forwarded_port", guest: 1080,  host: 1080,  auto_correct: true   # Mailcatcher
```

Comment:
Why are you ignoring the previous logic here? It says, that there were some problems on Windows to forward ports. Before removing, it should be tested.